spec_util/matcher.go

```diff
@@ -10,15 +11,39 @@ import (
 )
 
 type methodRegexp struct {
-	Operation string         // HTTP operation
-	Host      string         // HTTP host
-	Template  string         // original method template
-	RE        *regexp.Regexp // template converted to regexp on path
+	Operation         string         // HTTP operation
+	Host              string         // HTTP host
+	Template          string         // original method template
+	RE                *regexp.Regexp // template converted to regexp on path
+	VariablePositions []int          // positions of path variables in templates
```

Comment:
Yes

Reply:
Add that to the comment?  Also consider adding a `newMethodRegexp` constructor that does the sorting (although that might be unnecessary since it looks like the input is already sorted based on how the list is created)?